docs/user/troubleshooting/plasma.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Plasma Troubleshooting
+summary: Troubleshooting Issues With the Plasma Desktop
+---
+
+# Plasma Desktop Troubleshooting
+
+This contains fixes for various issues you might encounter with the Plasma desktop environment.
+
+## Desktop no longer responds to clicks, cursor stuck in "move" mode (cross)
+
+This is often caused by a Firefox bug. It can be triggered by various things, like moving Firefox windows.
+
+**Symptoms:**
```

Comment:
I don't want to make the symptoms headings TOC headings, which is why I didn't use the h3 formatting. Otherwise, in a TOC you would see

Plasma Desktop Troubleshooting
 Desktop no longer responds to clicks, cursor stuck in "move" mode (cross)
  Symptoms
 Application windows look corrupt / incomplete
  Symptoms

etc. "Symptoms" is not valuable as a header item.